.circleci/config.yml

```diff
@@ -251,17 +251,23 @@ jobs:
       - run:
           name: Pull OWASP ZAP docker image
           command: docker pull owasp/zap2docker-weekly
+      - run:
+          name: Make reports directory group writeable
+          command: chmod g+w reports
       - run:
           name: Run OWASP ZAP
           command: |
             docker run \
-              -v $(pwd)/zap.conf:/zap/wrk/zap.conf:rw \
+              -v $(pwd)/zap.conf:/zap/wrk/zap.conf:ro \
               -v $(pwd)/reports:/zap/wrk:rw \
               --rm \
+              --user zap:$(id -g) \
```

Comment:
Use the zap user that the docker container is expecting to run the scan, but assign them the same group id as the host user has, to allow writing out the html report to the /reports host directory.